apps/daimo-mobile/src/view/screen/onboarding/OnboardingScreen.tsx

```diff
@@ -77,7 +75,7 @@ export default function OnboardingScreen({
   // During onboarding, listen for payment or invite link invites
   useEffect(() => {
     getInitialURLOrTag(true).then((url) => {
-      if (url == null) return;
+      if (!url) return;
```

Comment:
intentional change?

`!url` is a slightly confusing statement roughly equivalent to `url == null || url === ""` = null, undefined, or ""

Reply:
ah not intentional; reverted